src/api/app/views/webui2/webui/comment/_content.html.haml

```diff
@@ -5,7 +5,7 @@
     %p
       = link_to(comment.user, user_show_path(comment.user))
       wrote
-      %u{ title: l(comment.created_at.utc) }
+      = link_to("#comment-#{comment.id}", title: l(comment.created_at.utc), name: "comment-#{comment.id}") do
```

Comment:
I find it a bit arbitrary to have the link on the timestamp.

What do you think about adding an icon next to the timestamp, like `%i.fas.fa-link.fa-sm{ title: 'Link to comment' }`? It would make this link a lot more obvious. I'm not so sure about the title, maybe it's not needed. Clicking the icon could also copy the link to the clipboard.

It would look like this:
![suggestion](https://user-images.githubusercontent.com/1102934/52072273-fd793280-2584-11e9-9d21-835a5bcd21a7.png)

Reply:
I find the icon a bit to noisy. For pages with many comments that would be a lot of icons.

I think users will figure out that this is a link and what it does. Just like they do with GitHub's coments.

Reply:
I find the underlining hint enough that it's a link - underlining text that is *not* a link is actually very confusing and I wonder why it was done this way.

Reply:
I do it in Github all the time, I like it as it is.

Reply:
> I find the icon a bit to noisy. For pages with many comments that would be a lot of icons.
> 
> I think users will figure out that this is a link and what it does. Just like they do with GitHub's coments.

noisy and why just there and not in all other links.. anything special about this link?